README.md

```diff
@@ -14,24 +14,19 @@
 
-However, implementing advanced multi-tenancy scenarios, for example, a private or public _Container-as-a-Service_ platform, it becomes soon complicated because of the flat structure of Kubernetes namespaces. In such scenarios, different groups of users get assigned a pool of namespaces with a limited amount of resources (e.g.: _nodes_, _vCPU_, _RAM_, _ephemeral and persistent storage_). When users need more namespaces or move resources from one namespace to another, they always need the intervention of the cluster admin because each namespace still works as an isolated environment. To work around this, and not being overwhelmed by continuous users' requests, cluster admins often choose to create multiple smaller clusters and assign a dedicated cluster to each organization or group of users leading to the well know and painful phenomena of the _clusters sprawl_.
+However, implementing advanced multi-tenancy scenarios, it becomes soon complicated because of the flat structure of Kubernetes namespaces. To overcome this, different groups of users or teams get assigned a dedicated cluster. As your organization grows, the number of clusters to manage and to keep aligned becomes a pain, leading to the well know phenomena of the _clusters sprawl_.
 
-**Capsule** takes a different approach. It aggregates multiple namespaces assigned to an organization or group of users in a lightweight abstraction called _Tenant_. Within each tenant, users are free to create their namespaces and share all the assigned resources between the namespaces of the tenant. The _Network and Security Policies_, _Resource Quota_, _Limit Ranges_, _RBAC_, and other constraints defined at the tenant level are automatically inherited by all the namespaces in the tenant leaving the tenant's users to freely allocate resources without any intervention of the cluster administrator.
+**Capsule** takes a different approach. In a single cluster, it aggregates multiple namespaces assigned to a team or group of users in a lightweight abstraction called _Tenant_. Within each tenant, users are free to create their namespaces and share all the resources in the tenant. The _Network and Security Policies_, _Resource Quota_, _Limit Ranges_, _RBAC_, and other constraints defined at the tenant level are automatically inherited by all the namespaces in the tenant. And users are free to admin their tenants in authonomy, without the intervention of the cluster administrator.
 
 # Use cases for Capsule
-Please, refer to the corresponding [section](use_cases.md) for a more detailed list of use cases that Capsule can address.
+Please, refer to the corresponding [section](documentation.md) in the documentation for a more detailed list of use cases that Capsule can address.
```

Comment:
Are you referring to `use-case.md` rather than `documentation.md`?

I don't know if you missed pushing this file or not.